schema/course.schema.json

```diff
@@ -80,11 +81,53 @@
               "title": "Hide the home button on the menu",
               "default": false
             },
+            "_hideBackButton": {
```

Comment:
The `.nav__back-btn` button isn't displayed on the menu (as part of core Adapt). I think you'd only be enabling the homeButton on the menu if you were redirecting to a start page.
In which case, should we remove `_hideBackButton` from the course level schema? This would also apply to [properties.schema](https://github.com/cgkineo/adapt-homeButton/blob/e60d67f0ab171a41e58bc46370c90d6861c478a2/properties.schema#L97).

Reply:
Thanks, @kirsty-hames . Yep, this makes sense to me. I've removed this in [978e4d7](https://github.com/cgkineo/adapt-homeButton/pull/27/commits/978e4d793d92f162c621d0226db085100b727815).